apps/bot/package.json

```diff
@@ -6,6 +6,7 @@
   "scripts": {
     "build": "tsc",
     "dev": "nodemon -e js,mjs,cjs,ts,mts,cts,json --ignore 'dist/**/*' --exec 'rimraf dist && tsc && node dist/main.js'",
+    "dev:win": "nodemon -e js,mjs,cjs,ts,mts,cts,json --ignore \"dist/**/*\" --exec \"rimraf dist && tsc && node dist/main.js\" ",
```

Comment:
```suggestion
    "dev":  "nodemon -e js,mjs,cjs,ts,mts,cts,json --ignore \"dist/**/*\" --exec \"rimraf dist && tsc && node dist/main.js\"",
```

Reply:
This should also work on other systems, so a separate script is not really needed.